Client/core/CMainMenu.h

```diff
@@ -97,6 +97,9 @@ class CMainMenu
 
     CGUI* m_pManager;
 
+    std::array<std::unique_ptr<CGUIStaticImage>, 6> m_socialsImages{};
+    std::map<const char*, const char*> m_socialMediaLinks;
```

Comment:
Seems like this is unused. (But you've shadowed it by `static const std::array<std::pair<std::string, std::string>, 6> m_socialMediaLinks`)
You can get rid of this.

Reply:
done